src/methods.ts

```diff
@@ -294,6 +444,7 @@ export type ChatPostEphemeralArguments = TokenOverridable & {
   user: string;
   as_user?: boolean;
   attachments?: MessageAttachment[];
+  blocks?: Block[];
```

Comment:
if we want to use `(KnownBlock | Block)[]` in all these places, maybe its worth giving that a type alias. so maybe its `type Block = KnownBlock | OtherBlock` (where `type KnownBlock = ImageBlock | ContextBlock | ...`, and `interface UnknownBlock { type: string; }`.